rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import { terser } from "rollup-plugin-terser";
+
+export default [
+  {
+    input: 'src/polyfill.js',
+    output: [
+      {
+        file: 'dist/polyfill.js',
+        format: 'umd'
```

Comment:
I'm still not convinced anyone actually wants `umd`, but there's probably no harm in it.

I guess I am a bit confused why `umd` doesn't get the document setup stuff that's in index.